dali/pipeline/operators/reader/coco_reader_op.cc

```diff
@@ -309,6 +309,12 @@ Tensor (`m` * `[x, y, w, h] or `m` * [left, top, right, bottom]`) and labels as
   .AddOptionalArg("ratio",
       R"code(If true, bboxes returned values as expressed as ratio w.r.t. to the image width and height. Default: False)code",
       false)
+  .AddOptionalArg("save_img_ids",
+      R"code(If true, image IDs will also be returned. Default: False)code",
+      false)
+  .AdditionalOutputsFn([](const OpSpec& spec) {
+    return static_cast<int>(spec.GetArgument<bool>("img_ids"));
```

Comment:
```suggestion
    return static_cast<int>(spec.GetArgument<bool>("save_img_ids"));
```
Check if this change works with coco_reader.ipynb or detection_pipeline.ipynb examples.